go/test/endtoend/vtgate/queries/aggregation/aggregation_test.go

```diff
@@ -452,3 +452,18 @@ func TestMinMaxAcrossJoins(t *testing.T) {
 		`SELECT /*vt+ PLANNER=gen4 */ t1.name, max(t1.shardKey), t2.shardKey, min(t2.id) FROM t1 JOIN t2 ON t1.t1_id != t2.shardKey GROUP BY t1.name, t2.shardKey`,
 		`[[VARCHAR("name 2") INT64(2) INT64(10) INT64(1)] [VARCHAR("name 1") INT64(1) INT64(10) INT64(1)] [VARCHAR("name 2") INT64(2) INT64(20) INT64(2)] [VARCHAR("name 1") INT64(1) INT64(20) INT64(2)]]`)
 }
+
+func TestComplexAggregation(t *testing.T) {
```

Comment:
@frouioui do we need to check the version of vtgate so these are not run on the upgrade/downgrade testing?